ui/app/AppLayouts/Profile/views/SettingsContentBase.qml

```diff
@@ -90,21 +100,57 @@ Item {
     }
 
     ScrollView {
-        anchors.top: titleRow.visible? titleRow.bottom : topHeader.bottom
+        id: scrollView
+        anchors.top: titleRow.visible ? titleRow.bottom : topHeader.bottom
```

Comment:
That's usually a sign layout or positioners should be used. This condition may become very complex (I've seen that a lot in our codebase) in the next iterations when some other items are added above ScrollView.

```qml
ColumnLayout {
   Item { id: topHeader }
   Item { id: anotherItem }
   ScrollView { id: scrollView }
}
```
vs
```qml
Item {
   Item { id: topHeader }
   Item { id: anotherItem; anchors.top: topHeader.visible ? topHeader.bottom : undefined}
   ScrollView { id: scrollView; anchors.top: anotherItem.visible ? anotcherItem.bottom : topHeader.visible ? topHeader.bottom : undefined }
}
```

Reply:
> Totally agree, but this is the way it was done when I found it :slightly_smiling_face:
> I tried not to refactor much to minimize the number conflicts...

Sure, that's understandable. I've just seen this line change and wanted to raise some attention/awareness. Leave it as is, it's not a major issue for now.

> Should we open an issue on this?

I don't think it requires a separate issue.